src/components/AddItemForm.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import fb from '../lib/firebase';
+import '../css/AddItemForm.css';
+import { v4 as uuidv4 } from 'uuid';
+
+const Form = () => {
+
+    const [itemName, setName] = useState("");
```

Comment:
It's a good idea to have these match:  `const [itemName, setItemName]`. If you do change it, make sure you change it wherever you are using it, also.